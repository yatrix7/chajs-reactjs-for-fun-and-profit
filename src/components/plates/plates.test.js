import React from 'react'
import renderer from 'react-test-renderer'
import Plates from './plates'

it('renders correctly with weight', () => {
	const tree = renderer
		.create(
			<Plates weight={355} className={'point-break-is-a-great-movie'} />
		)
		.toJSON()
	expect(tree).toMatchSnapshot()
})

it('renders correctly with no weight', () => {
	const tree = renderer
		.create(
			<Plates weight={0} className={'point-break-is-a-great-movie'} />
		)
		.toJSON()
	expect(tree).toMatchSnapshot()
})
