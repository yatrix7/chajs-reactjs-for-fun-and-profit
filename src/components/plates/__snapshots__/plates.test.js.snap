// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with no weight 1`] = `
<span
  className="plates point-break-is-a-great-movie"
>
  Plates per side: 
  0
</span>
`;

exports[`renders correctly with weight 1`] = `
<span
  className="plates point-break-is-a-great-movie"
>
  Plates per side: 
  45: 3, 10: 2
</span>
`;
